src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -79,6 +79,8 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):#pylint:
 register_cli_argument('acs', 'container_service_name', options_list=('--name', '-n'), help='The name of the container service', completer=get_resource_name_completion_list('Microsoft.ContainerService/ContainerServices'))
 register_cli_argument('acs create', 'agent_vm_size', completer=get_vm_size_completion_list)
 register_cli_argument('acs scale', 'new_agent_count', type=int, help='The number of agents for the cluster')
+register_cli_argument('acs create', 'service_principal', help='Service principal for making calls into Azure APIs')
+register_cli_argument('acs create', 'client_secret', help='Client secret to use with the service principal for making calls to Azure APIs')
```

Comment:
For your consideration, azure-cli commands use `password` terminology across for this concept. For consistency I would suggest you use the `password` here. Both terminologies are correct, but have used at different layers. 
At this point it is your call, because you have to update related method as well